### PR TITLE
Feature #74 implemented shippingAddress interface via REST

### LINKED ISSUE #74: Implement ShippingAddress interface via REST

### PR DESCRIPTION
**Pull request by kushnir-tadey**
closes #74

### CODE REVIEW

File: ProjectShop/ProjectShop/router.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from rest_framework import routers
 from order.views import OrderViewSet
```
Comment (MykhailoGrynenko) [inline ProjectShop/ProjectShop/router.py:2]: Please, add a blank line according to PEP 8

File: ProjectShop/Shipping/serializers.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from Shipping.models import ShippingModel
```
Comment (MykhailoGrynenko) [inline ProjectShop/Shipping/serializers.py:2]: Please, add a blank line according to PEP 8

File: ProjectShop/Shipping/views.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from rest_framework import viewsets, renderers
```
Comment (MykhailoGrynenko) [inline ProjectShop/Shipping/views.py:1]: Have you used renderers?

File: ProjectShop/Shipping/views.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+from rest_framework.decorators import action
+from rest_framework.response import Response
```
Comment (MykhailoGrynenko) [inline ProjectShop/Shipping/views.py:3]: Have you used Response?

File: ProjectShop/Shipping/views.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+from Shipping.models import ShippingModel
+
```
Comment (MykhailoGrynenko) [inline ProjectShop/Shipping/views.py:6]: A blank line is not needed here.

**Review by MykhailoGrynenko — APPROVED** [review]
LGTM

**Review by v-lavrushko — APPROVED** [review]

**Review by ProkhvatylovArtem — APPROVED** [review]

**Review by strangerok1507 — APPROVED** [review]

**Pull request merged**